docs/builtins/Keysets/enforce-keyset.md

```diff
@@ -0,0 +1,40 @@
+## enforce-keyset
+Use `enforce-keyset` to execute a specified `GUARD` or a defined keyset named `KEYSETNAME` to enforce the desired predicate logic.
+
+### Basic syntax
+
+To execute a `GUARD` or a defined keyset to enforce desired predicate logic, use the following syntax:
+
+`(enforce-keyset GUARD)`
+`(enforce-keyset KEYSETNAME)`
+
+### Arguments
+
+Use the following arguments to specify the `GUARD` or `KEYSETNAME` for the `enforce-keyset` Pact function:
+
+| Argument   | Type   | Description                                        |
+|------------|--------|----------------------------------------------------|
+| `GUARD`    | `guard`  | Specifies the guard to execute.                    |
+| `KEYSETNAME` | `string` | Specifies the name of the defined keyset to enforce.|
+
+### Return values
+
+The `enforce-keyset` function returns a boolean value indicating whether the guard or keyset enforced the desired predicate logic.
```

Comment:
```suggestion
The `enforce-keyset` function returns a boolean value indicating whether the guard or keyset was enforced successfully.
```

Reply:
Also, looking at the implementation, at least some code paths seem to give a `VError` instead of returning a `False` as either the original or my suggested sentence my imply. Do we want to mention this somehow?